bee-api/bee-rest-api/src/endpoints/config.rs

```diff
@@ -3,10 +3,10 @@
 
 use serde::Deserialize;
 
-use std::net::{IpAddr, Ipv4Addr, Ipv6Addr, SocketAddr};
+use std::net::{IpAddr, Ipv4Addr, Ipv6Addr, SocketAddr, ToSocketAddrs};
+use libp2p::{multiaddr::Protocol, Multiaddr};
```

Comment:
just a little thing, but please order the imports in following way:

1) bee-rest-api imports
2) workspace dependencies
3) external dependencies
4) std dependencies

Reply:
With a blank line between sections!